assets/base.css

```diff
@@ -2291,6 +2291,15 @@ input[type='checkbox'] {
 .header__heading-logo {
   height: auto;
   width: 100%;
+  transition: width var(--duration-short) ease;
+}
+
+.scrolled-past-header .header--reduce-25 .header__heading-logo {
+  width: 75%;
+}
+
+.scrolled-past-header .header--reduce-50 .header__heading-logo {
+  width: 50%;
```

Comment:
I don't think we can use `transform` here because we need to reduce the actual area the logo will occupy, as the primary reason to reduce the logo on scroll is to make the sticky bar shorter.

`transform` simply resizes the image but takes the same space.

I can tweak the animation speed but I'm not sure we'll be able to avoid animating width 👀

Reply:
Seems like on Express we did it by using the `max-width` and setting the `width` to either `100%` when the header is at the top and `auto` when sticky. 🤔 And Im not having an issue on my phone in terms of the animation.